staramr/detection/AMRDetection.py

```diff
@@ -97,8 +101,38 @@ def run_amr_detection(self, files, pid_threshold, plength_threshold_resfinder, p
         :param plength_threshold_pointfinder: The percent length overlap for BLAST results (pointfinder).
         :param plength_threshold_plasmidfinder: The percent length overlap for BLAST results (plasmidfinder).
         :param report_all: Whether or not to report all blast hits.
+        :param ignore_invalid_files: Skips the invalid input files if set.
         :return: None
         """
+
+        total_files = len(files)
+        invalid_files = 0
+        removeable_files = []
+
+        for file in files:
+            try:
+                # Tries to return a single valid sequence, will raise an error if the input is incorrect or has more than one
+                SeqIO.read(file, "fasta")
+            except ValueError as e:
+                if str(e) != "More than one record found in handle":
+                    if ignore_invalid_files:
+                        logger.info('--ignore-invalid-files is set, skipping file {}'.format(file))
+                        invalid_files +=1
+                        removeable_files.append(file)
+                    else:
+                        raise AssertionError('File {} is invalid, please use --ignore-invalid-files to skip over invalid input files'.format(file))
```

Comment:
I would just raise an `Exception` here. I think `AssertionError` is primary to be used with Python `assert` statements.

Reply:
I agree, I fixed this in 25f8137